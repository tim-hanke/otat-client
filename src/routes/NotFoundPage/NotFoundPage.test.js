import React from "react";
import ReactDOM from "react-dom";
import { MemoryRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import NotFoundPage from "./NotFoundPage";

describe("NotFoundPage component", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <MemoryRouter>
        <NotFoundPage />
      </MemoryRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <NotFoundPage />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
