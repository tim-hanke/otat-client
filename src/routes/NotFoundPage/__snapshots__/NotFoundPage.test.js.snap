// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders the UI as expected 1`] = `
<section
  className="Section NotFoundPage"
>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</section>
`;
